test/1_test_fullpath.js

```diff
@@ -166,181 +1023,100 @@ contract('Voucher tests', async (addresses) => {
         internalTx,
         'LogVoucherDelivered',
         (ev) => {
-          return ev._issuer === users.seller.address;
+          tokenVoucherKey1 = ev._tokenIdVoucher;
+          return ev._tokenIdVoucher.gt(new BN(0));
         },
         'order1 not created successfully'
       );
 
-      const filtered = internalTx.logs.filter(
-        (e) => e.event === 'LogVoucherDelivered'
-      )[0];
-
-      tokenVoucherKey1 = filtered.returnValues['_tokenIdVoucher'];
+      //Buyer redeems voucher
+      await contractBosonRouter.redeem(tokenVoucherKey1, {
+        from: users.buyer.address,
+      });
     });
 
-    it('fill second order (aka commit to buy a voucher)', async () => {
-      const txFillOrder = await contractBosonRouter.requestVoucherETHETH(
-        tokenSupplyKey2,
-        users.seller.address,
-        {
-          from: users.buyer.address,
-          to: contractBosonRouter.address,
-          value: constants.PROMISE_PRICE2 + constants.PROMISE_DEPOSITBU2,
-        }
+    it('withdraw the escrowed payment from one redeemed voucher', async () => {
+      const buyerEscrowedBefore = await contractCashier.getEscrowAmount.call(
+        users.buyer.address
       );
-      const internalTx = await truffleAssert.createTransactionResult(
-        contractVoucherKernel,
-        txFillOrder.tx
+
+      const sellerBalanceBefore = await new BN(
+        await web3.eth.getBalance(users.seller.address)
       );
 
+      console.log('sellerBalanceBefore ', sellerBalanceBefore.toString());
```

Comment:
Do we need this `console.log` ?

Reply:
Oops. Nope. Will be removed in next commit.